src/test/java/org/minions/devfund/yuri/linkedlist/InsertSortedTest.java

```diff
@@ -0,0 +1,169 @@
+                + "list", var1.size() == LISTSIZE1);
+        Assert.assertTrue("insertSorted does not correctly insert element when placing value at end of list",
+                (Integer) var1.get(0) == LISTFIRSTVALUE);
+        Assert.assertTrue("insertSorted does not correctly insert element when placing value at end of list",
+                (Integer) var1.get(1) == LISTSECONDVALUE);
+        Assert.assertTrue("insertSorted does not correctly insert element when placing value at end of list",
+                (Integer) var1.get(2) == LISTTHIRDVALUE);
+        Assert.assertTrue("insertSorted does not correctly insert element when placing value at end of list",
+                (Integer) var1.get(LISTFIRSTVALUE) == LISTNEWVALUE2);
+    }
+
+    /**
+     * test.
+     */
+    @Test
+    public void testInsertFront() {
+        LinkedList var1 = new LinkedList();
+        var1.add(LISTFIRSTVALUE);
+        var1.addLast(LISTSECONDVALUE);
+        var1.addLast(LISTTHIRDVALUE);
+
+        try {
+            LinkedListUtils.insertSorted(var1, LISTNEWVALUE3);
+        } catch (Exception var3) {
+            Assert.fail("insertSorted throws " + var3 + " when placing value at front of list");
+        }
+
+        Assert.assertTrue("insertSorted creates LinkedList of incorrect size when placing value at front of "
+                + "list", var1.size() == LISTSIZE1);
+        Assert.assertTrue("insertSorted does not correctly insert element when placing value at front of "
+                + "list", (Integer) var1.get(0) == LISTNEWVALUE3);
+        Assert.assertTrue("insertSorted does not correctly insert element when placing value at front of "
+                + "list", (Integer) var1.get(1) == LISTFIRSTVALUE);
+        Assert.assertTrue("insertSorted does not correctly insert element when placing value at front of "
+                + "list", (Integer) var1.get(2) == LISTSECONDVALUE);
+        Assert.assertTrue("insertSorted does not correctly insert element when placing value at front of "
+                + "list", (Integer) var1.get(LISTSIZE2) == LISTTHIRDVALUE);
+    }
+
+    /**
+     * test.
+     */
+    @Test
+    public void testInsertMiddleEqualToExistingElement() {
+        LinkedList var1 = new LinkedList();
+        var1.add(LISTFIRSTVALUE);
+        var1.addLast(LISTSECONDVALUE);
+        var1.addLast(LISTTHIRDVALUE);
+
+        try {
+            LinkedListUtils.insertSorted(var1, LISTSECONDVALUE);
+        } catch (Exception var3) {
+            Assert.fail("insertSorted throws " + var3 + " when placing value that is equal to value already in list");
+        }
+
+        Assert.assertTrue("insertSorted creates LinkedList of incorrect size when placing value that is "
+                + "equal to value already in list", var1.size() == LISTSIZE1);
+        Assert.assertTrue("insertSorted does not correctly insert element when placing value that is equal "
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "insertSorted does not correctly insert element when placing value that is equal " 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)

Reply:
done